src/communication_managers/iothub_communication_manager/src/iothub_communication_manager.c

```diff
@@ -440,7 +440,7 @@ static bool ADUC_DeviceClient_Create(
  * @param connectionString the connection string to scan
  * @returns the connection type for @p connectionString
  */
-static ADUC_ConnType GetConnTypeFromConnectionString(const char* connectionString)
```

Comment:
If these are not used outside of iothub_communication_manager.c, then let's keep the static keywords so that they are not externally linked.